example/lib/tweet_page.dart

```diff
@@ -26,11 +26,18 @@ class TweetPage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
+      backgroundColor: Colors.grey[900],
       appBar: AppBar(
         title: Text(mediaType),
       ),
       body: ListView(
         children: <Widget>[
+          buildHeader("$mediaType TweetEmbed"),
+          buildTweetEmbed(tweetPath),
+          Divider(height: 80,),
+          buildHeader("$mediaType Quote TweetEmbed"),
+          buildTweetEmbed(quoteTweetPath),
+          Divider(height: 80,),
```

Comment:
Maybe smaller dividers? And add the dividers to all sections or tweets, to be consistent.